### ISSUE TITLE
Baseline-RGCN

### ISSUE THREAD

**Issue by lujiaying** [lujiaying/Open-World-TaxoKG-CoLearning#19]
Paper: https://arxiv.org/pdf/1703.06103.pdf
dgl implementation: https://docs.dgl.ai/en/0.4.x/tutorials/models/1_gnn/4_rgcn.html

Scripts to mimic:
- [model/CompGCN](https://github.com/lujiaying/Learn-to-Abstract-Taxonomy/blob/master/model/CompGCN.py)
    - CompGCN use BCE loss, but RGCN use cross-entropy loss. This might lead significant script changes
- [scripts/train_CompGCN](https://github.com/lujiaying/Learn-to-Abstract-Taxonomy/blob/master/scripts/train_CompGCN.py)
    - config [L32](https://github.com/lujiaying/Learn-to-Abstract-Taxonomy/blob/master/scripts/train_CompGCN.py#L32), main run [L155](https://github.com/lujiaying/Learn-to-Abstract-Taxonomy/blob/master/scripts/train_CompGCN.py#L155)
    - node/edge embs are first obtained through token-encoder, then feed into graph neural network, as can be seen in L219-L224

Working Directory on Turing: `/local/home/jlu229/Learn-to-Abstract-Taxonomy`; venv: `conda activate taxoKG`
